docs/api-publishing/03-OAS.md

```diff
@@ -0,0 +1,267 @@
+
+### Paths Section
+
+The `paths` section is a parent property that contains a list of the resource paths in the API and the properties associated with each resource path.
+
+|Property|Description|Requirement|
+|:---|:---|:---|
+|`{path}.summary`| A short human readable summary of the resource purpose | **MUST** |
+|`{path}.description`|See [Property Descriptions](#property-descriptions)|**MUST**|
+|`{path}.{http-verb}.summary`|A short human readable summary of the resource operation purpose|**MUST**|
+|`{path}.{http-verb}.operationId`|A **UNIQUE** string to identify the operation. Used by tools and libraries to uniquely identify the operation.|**MUST**|
+|`{path}.{http-verb}.requestBody`|The request body appropriate for this operation. See [Request Body](#request-body).|**MUST** for `POST`, `PUT`, `PATCH` verbs. **MUST NOT** for `GET`, `DELETE`, `HEAD`, `OPTIONS` verbs.|
+|`{path}.{http-verb}.responses`|A list of the responses that an API Consumer can expect from the operation. See [Responses](#responses)|**MUST**|
+
+
+#### Responses
+
+`{path}.{http-verb}.responses`
+
+An open API specification path/http-verb **SHOULD** include **ALL** responses by HTTP response code. **MUST** include error responses and where possible **SHOULD** refer to an error schema
+
+|Property|Description|Requirement|
+|:---|:---|:---|
+|`{path}.{http-verb}.responses.{code}.description`|See [Property Descriptions](#property-descriptions)|**MUST**|
+|`{path}.{http-verb}.responses.{code}.content`|The definition of the content that an API Consumer can expect when consuming the API|**MUST**|
+|`{path}.{http-verb}.responses.{code}.{media-type}`|The media type(s) that the API COnsumer can expect in the response - e.g. `application/fhir+json`. See also [Accept Headers](../api-development/Synchronous%20APIs/Headers#accept)|**MUST**|
+|`{path}.{http-verb}.responses.{code}.{media-type}.schema`|The schema definition associated with the response|**MUST**|
+
+```yaml
+responses:
+  '200':
+    description: API request success
+    content:
+      application/fhir+json:
+        schema:
+          $ref: '#/components/schemas/Observation'
+  '400':
+    description: Bad request error. The request failed due to invalid input format or a validation error - retrying the request will not succeed
+    content:
+      application/fhir+json:
+        schema:
+          $ref: '#/components/schemas/OperationOutcome'
+  '404':
+    description: Resource not found error. The request failed as the requested resource could not be found on the server.
+    content:
+      application/fhir+json:
+        schema:
+          $ref: '#/components/schemas/OperationOutcome'
+  '500':
+    description: Server error
+    content:
+      application/fhir+json:
+        schema:
+          $ref: '#/components/schemas/OperationOutcome'
+```
+
+
+API Providers **SHOULD NOT** include a `default` response. Whilst supported by OpenAPI this is discouraged.
```

Comment:
Good question - this is generally discouraged however there are situations where it might be appropriate. For example a FHIR API might have a `default` response for all error scenarios that refers to an `OperationOutcome` schema where various examples are covered such as `404`, `400`, `429` etc. Generally though it is considered bast practice to document, per resource/verb the responses that an API consumer can expect from the API